packages/amplication-server/src/core/entity/entity.service.ts

```diff
@@ -569,20 +526,20 @@ export class EntityService {
     });
 
     if (existingEntities.length > 0) {
-      existingEntities.forEach((entity) => {
-        log.push(
-          new ActionLog({
-            message: `Entity "${entity.name}" already exists`,
-            level: EnumActionLogLevel.Error,
-          })
-        );
+      await Promise.all(
+        existingEntities.map(async (entity) => {
+          void actionContext.logByStep(
+            EnumActionLogLevel.Error,
+            `Entity "${entity.name}" already exists`
+          );
```

Comment:
can't you avoid the `Promise.all()` and the `async` since you added the `void` and treat everything as fire-and-forget?

Reply:
You are right. I forgot to undo it after we changed it